custom_components/huesensor/remote.py

```diff
@@ -347,7 +342,14 @@ def unique_id(self):
     def state(self):
         """Return the state of the sensor."""
         data = self._data.get(self._hue_id)
-        if data and data["changed"]:
```

Comment:
@robmarkcole Even if you do not proceed with this PR, you should probably remove the "data[changed]" test, or the remotes have an "unknown" state most of the time

Reply:
ok